$OpenBSD: patch-build_make_configure_sh,v 1.8 2015/02/11 00:07:47 brad Exp $

https://chromium.googlesource.com/webm/libvpx/+/769fe6e2b38d9dbb8b60f3af9e0555614d77caf8

configure: enable x86inc for all intel platforms

there are no known issues since:
10d5e09 Fix issues in 32bit PIC enabled build

related issues: #808, #924

--- build/make/configure.sh.orig	Thu Feb  5 20:04:43 2015
+++ build/make/configure.sh	Thu Feb  5 20:06:41 2015
@@ -1198,9 +1198,7 @@ EOF
         fi
     fi
 
-    # default use_x86inc to yes if pic is no or 64bit or we are not on darwin
-    echo "  checking here for x86inc \"${tgt_isa}\" \"$pic\" "
-    if [ ${tgt_isa} = x86_64 -o ! "$pic" = "yes" -o "${tgt_os#darwin}" = "${tgt_os}"  ]; then
+    if [ "${tgt_isa}" = "x86_64" ] || [ "${tgt_isa}" = "x86" ]; then
       soft_enable use_x86inc
     fi
 
